sorting/heap_sort.cpp

```diff
@@ -47,6 +47,4 @@ int main() {
     for (int i = 0; i < n; ++i) {
         std::cout << a[i] << std::endl;
     }
-
-    std::getchar();
```

Comment:
Why remove this?

Reply:
std::getchar() build failed.
so I'm just remove it.

Reply:
You can use `std::cin.get()`.

Reply:
thank you again